Apps/WebClient/src/ClientApp/src/views/dependents.vue

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+import { ILogger } from "@/services/interfaces";
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import container from "@/plugins/inversify.config";
+import LoadingComponent from "@/components/loading.vue";
+
+@Component({
+    components: {
+        LoadingComponent,
+    },
+})
+export default class DependentsView extends Vue {
+    private logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
+    private isLoading = true;
+    private hasErrors = false;
+    private errorMessage = "";
+
+    private handleError(error: string): void {
+        this.hasErrors = true;
+        this.errorMessage = error;
```

Comment:
error message and has errors not necessary

Reply:
removed